lib/src/rules/avoid_js_rounded_ints.dart

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2018, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/dart/ast/visitor.dart';
+import 'package:linter/src/analyzer.dart';
+
+const _desc = r'Avoid JavaScript rounded ints.';
+
+const _details = r'''
+
+**AVOID** integers that will be rounded.
```

Comment:
Perhaps "integers" --> "integer literals"

Reply:
done

Reply:
This is more precise: **AVOID** integer literals that cannot be represented exactly when compiled to JavaScript.

Reply:
done